easybuild/easyconfigs/g/GAP/GAP-2.5.0.eb

```diff
@@ -20,8 +20,8 @@ checksums = ['28cf96bedb9550f20b32d340c7572ac2d15ac4aa94bfc6f33d1b755767f7f7bd']
 # Dependencies.
 #
 dependencies = [
-    ('Molgenis-Compute', 'v17.08.1', '-Java-1.8.0_74'),
-    ('BeadArrayFiles', '1.3.1','-Python-2.7.11', ('foss','2015b')),
+    ('Molgenis-Compute', 'v19.01.1', '-Java-11-LTS'),
+    ('BeadArrayFiles', '1.3.1','-Python-2.7.16', ('foss','2018b')),
```

Comment:
BeadArrayFiles needs PythonPlus for a single Python package dependency ```numpy```. Unfortunately that is one that does use OpenBLAS and hence the 2018b version can produce wrong results when doing vector computations due to the bug in the old OpenBLAS. If it is problematic it doesn't mess with enough SNVs to mess up the concordance check as we would have noticed when a lot of them are wrong. But there is no guarantee all individual values for the SNVs on the arrays are correct, so this would be a candidate for an update. Morover Python 2.x is problematic by now, but the BeadArrayFiles repo mentions explicitly:
`This library is compatible with Python 3 and was tested with Python 3.8.5`

Reply:
added a new version of BeadArrayFiles